mlir/lib/Pass/PassRegistry.cpp

```diff
@@ -6,8 +6,8 @@
 //
 //===----------------------------------------------------------------------===//
 
-#include <utility>
 #include <optional>
+#include <utility>
```

Comment:
Since you're touching these lines: can you move them after the project headers and move `#include "mlir/Pass/PassRegistry.h"` here instead?